.github/ISSUE_TEMPLATE/syncer-support.md

```diff
@@ -48,3 +48,5 @@ Additionally, please provide the following info before submitting your issue. Th
 3. Common issues
     - [ ] Is the `worker-count` set to a reasonable value (suggest-32)?
     - [ ] Is the `batch` set to a reasonable value (suggest 100-1000)?
+    - [ ] Is the downstream server MySQL and using `multiStatements` (syncer doesn't provide `multiStatements`)?
```

Comment:
remove it, user don't know what's `multiStatements`, and we can support `multiStatements` in syncer